### PR TITLE
Resource usage docs

### PR DESCRIPTION
**Pull request by jorgebay**
Document CPU and memory utilization on Kubernetes.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -207,8 +208,25 @@ To control whether the LogDNA agent collects Kubernetes events, configure the `L
...
+
+The agent is deployed as a Kubernetes DaemonSet, creating one pod per node selected. The agent collects logs of all
+the pods in the node. The resource requirements of the agent are in direct relation to the amount of pods per node,
+and the amount of logs producer per pod.
+
+The agent requires at least 128Mib and no more than 512Mib of memory. It requires at least
+[twenty millicpu (`20m`)][k8s-cpu-usage].
+
+Different features can also increase resource utilization. When line exclusion/inclusion or redaction rules
+are specified, you can expect to additional CPU consumption per line and per regex rule defined. When Kubernetes
```
Comment (jorgebay) [inline docs/README.md:225]: > per regex rule defined. When

Should this be a [semicolon](https://www.youtube.com/watch?v=M94ii6MVilw&ab_channel=thelonelyisland)?

File: docs/README.md [hunk 2]
```diff
@@ -207,8 +208,25 @@ To control whether the LogDNA agent collects Kubernetes events, configure the `L
 
 * `always` - Always capture events
 * `never` - Never capture events
-__Note:__ The default option is `always`.
+__Note:__ The default option is `never`.
```
Comment (jorgebay) [inline docs/README.md:211]: The default was incorrectly documented.

**Review by c-nixon — APPROVED** [review]
This reads well and all looks correct to me

File: docs/README.md (hunk 1 above)
Comment (alfeng6) [reply docs/README.md:225]: period looks fine

**Review by alfeng6 — APPROVED** [review]
looks great to me

**Pull request merged**